src/main/java/com/generation/greengen/model/Produto.java

```diff
@@ -43,6 +43,10 @@ public class Produto {
 	@JsonIgnoreProperties("produto")
 	private Categoria categoria;
 
+	@ManyToOne
+	@JsonIgnoreProperties ("usuarios")
```

Comment:
Aqui era para estar `@JsonIgnoreProperties ("produto")` ?
```suggestion
	@JsonIgnoreProperties ("produto")
```

Reply:
@Jonathangaruti